rai/dicom/uid.py

```diff
@@ -21,12 +21,12 @@
 # valid UIDs (http://www.medicalconnections.co.uk/FreeUID.html)
 # Their service was used to obtain the following root UID for
 # Radiotherapy AI:
-RAI_ROOT_UID = "1.2.826.0.1.3680043.10.756."
+_RAI_ROOT_UID = "1.2.826.0.1.3680043.10.756."
 
-RAI_INTERNAL_ROOT_UID = f"{RAI_ROOT_UID}0."
-RAI_CLIENT_ROOT_UID = f"{RAI_ROOT_UID}1."
+_RAI_INTERNAL_ROOT_UID = f"{_RAI_ROOT_UID}0."
 
-PRODUCT_ID = "000000"
+_PRODUCT_ID = "000000"
 
-RAI_IMPLEMENTATION_CLASS_UID = f"{RAI_INTERNAL_ROOT_UID}{PRODUCT_ID}.{__version__}"
+RAI_CLIENT_ROOT_UID = f"{_RAI_ROOT_UID}1."
+RAI_IMPLEMENTATION_CLASS_UID = f"{_RAI_INTERNAL_ROOT_UID}{_PRODUCT_ID}.{__version__}"
```

Comment:
suggest processing _PRODUCT_ID with something that eats leading zeros, or converts to an integer and then convert back to string representation of that integer (base 10).

Reply:
As an alternative, to verify that an appropriate UID has been created I opted to run it through pydicom's UID verification checker.

Fixed in [7f8e9bd](https://github.com/RadiotherapyAI/rai/pull/82/commits/7f8e9bd1c782a5851bc830f7dca39918eaf599c9)